features/facebookImport/src/model/analyses/report/no-data-folders.js

```diff
@@ -18,8 +15,8 @@ export default class NoDataFoldersAnalysis extends ReportAnalysis {
 
     async analyze({ zipFile }) {
         const entries = await relevantZipEntries(zipFile);
-        const extractedFolderNames = entries.map((fileName) => {
-            const fileNameWithoutId = removeEntryPrefix(fileName);
+        const extractedFolderNames = entries.map((entry) => {
+            const fileNameWithoutId = entry._path;
```

Comment:
This constant is probably not needed then. Possibly same applies to other cases.